bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Magnitude.scala

```diff
@@ -1,5 +1,7 @@
 package edu.gemini.spModel.core
 
+import scalaz._
+import Scalaz._
 
 case class Magnitude(value: Double, band: MagnitudeBand, error: Option[Double], system: MagnitudeSystem) {
 
```

Comment:
Not in your code but should we remove the old `Ordering`, it is misleading as you can't really sort Magnitudes of different bands